INSTALL/grub/help/en_US.txt

```diff
@@ -1,8 +1,8 @@
  L - Language Select
 F1 - Display this help information
```

Comment:
I am not sure about this one, I will take a look at it in the latest release and update here

```suggestion
F1 - Display help information
```

Reply:
Edit: Seems like the release was put on hold. So if this change makes sense feel free to commit it.